saleor/plugins/webhook/tasks.py

```diff
@@ -474,11 +484,70 @@ def send_webhook_request_sync(
 
     attempt_update(attempt, response)
     delivery_update(delivery, response.status)
+    report_event_delivery_attempt(attempt)
     clear_successful_delivery(delivery)
 
     return response_data if response.status == EventDeliveryStatus.SUCCESS else None
 
 
+@app.task
+def observability_send_events():
+    events = buffer.get_events()
+    if not events:
+        return 0
```

Comment:
It is celery task. Is it expected that you return some value here and below? `len(events)`?

Reply:
You are right. That was an artifact from an abandoned idea. Removed